utils/src/main/kotlin/FileMatcher.kt

```diff
@@ -38,22 +38,22 @@ class FileMatcher(
     /**
      * Toggle the case-sensitivity of the matching.
      */
-    caseSensitive: Boolean = true
+    ignoreCase: Boolean = false
```

Comment:
How about squashing this commit into "FileMatcher: Add an option for case-sensitivity" as you are fixing an issue you introduced in the same PR?

Reply:
I'd like to keep it `as-is` as I believe it's good enough to not spend any further time on it.